docs/en/tutorials/getting-started.rst

```diff
@@ -27,7 +27,7 @@ What is Doctrine?
 -----------------
 
 Doctrine ORM is an `object-relational mapper (ORM) <https://en.wikipedia.org/wiki/Object-relational_mapping>`_
-for PHP 7.1+ that provides transparent persistence for PHP objects. It uses the Data Mapper
+for PHP 8.1+ that provides transparent persistence for PHP objects. It uses the Data Mapper
```

Comment:
```suggestion
for PHP that provides transparent persistence for PHP objects. It uses the Data Mapper
```

Not sure we need a version number here.

Reply:
Changed with a amend / force push, is it the regular process?